pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -151,5 +177,732 @@ func CreateCustomResourceDefinitions(clientset apiextensionsclient.Interface) er
 		return err
 	}
 
+	log.Info("Creating v2.CiliumNetworkPolicy CustomResourceDefinition... DONE")
```

Comment:
Change to `log.Info("done creating v2.CiliumNetworkPolicy CustomResourceDefinition")`

Reply:
Done